pkg/koordlet/statesinformer/states_nodemetric.go

```diff
@@ -483,7 +482,7 @@ func (r *nodeMetricInformer) collectNodeGPUMetric(queryparam metriccache.QueryPa
 }
 
 func (r *nodeMetricInformer) collectNodeAggregateMetric(endTime time.Time, aggregatePolicy *slov1alpha1.AggregatePolicy) []slov1alpha1.AggregatedUsage {
-	aggregateUsages := []slov1alpha1.AggregatedUsage{}
+	var aggregateUsages []slov1alpha1.AggregatedUsage
```

Comment:
make([], )

Reply:
It's OK in Golang to append a nil slice.